Solutions/ReversingLabs/Data/Solution_ReversingLabs.json

```diff
@@ -1,12 +1,23 @@
 {
-    "Name": "ReversingLabs",
-    "Author": "Nikhil Tripathi - v-ntripathi@microsoft.com",
-    "Description": "[ReversingLabs](https://www.reversinglabs.com/) provides explainable threat intelligence into malware infected files and objects, for any file, any location, and any threat.",
-    "Playbooks": [
-      "Playbooks/Enrich-SentinelIncident-ReversingLabs-File-Information/azuredeploy.json"
-    ],
-	"Metadata": "SolutionMetadata.json",
-    "BasePath": "C:\\GitHub\\azure\\Solutions\\ReversingLabs",
-	"Version": "1.1.0"
-  }
-  
\ No newline at end of file
+  "Name": "ReversingLabs",
+  "Author": "ReversingLabs - support@reversinglabs.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/reversinglabs.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The ReversingLabs solution for Microsoft Sentinel includes a number of Sentinel resources designed to automate your security operations using the power of TitaniumCloud APIs and visualize your threat intelligence capabilities using included workbooks.",
+  "Workbooks": [ 
+	  "Workbooks\\ReversingLabs-CapabilitiesOverview\\ReversingLabs-CapabilitiesOverview.json"
+  ],
+  "Analytic Rules": [],
+  "Playbooks": [
+	  "Playbooks\\ReversingLabs-EnrichFilehash\\azuredeploy.json"
+  ],
+  "Parsers": [],
+  "SavedSearches": [],
+  "Hunting Queries": [],
+  "Data Connectors": [],
+  "Watchlists": [],
+  "WatchlistDescription": [],
+  "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\ReversingLabs",
+  "Version": "1.0.0",
+  "Metadata": "SolutionMetadata.json",
+  "TemplateSpec": false
```

Comment:
@AaronHoffmannRL  can please change  TemplateSpec value  true and please add one more property "Is1Pconnector": false.